WTWSMS/history/provinces/100 - Avranches.txt

```diff
@@ -22,8 +22,6 @@ religion = celtic_pagan
 terrain = forest
 # History
 1.1.1 = { # Viking takeover
-	remove_settlement = b_caen
-	remove_settlement = b_bayeux
```

Comment:
Yes, but my statement was that I don't get the block itself at all, given that the viking takeover itself is anachronistic at that date.